homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,75 @@
+"""GC-check function"""
+def GC_count_fun(f, gc_bounds = (0, 100)):
```

Comment:
В питоне все переменные (за исключением классов и глобальных констант) принято называть маленькими буквами. Даже если это аббревиатуры
```suggestion
def gc_count_fun(f, gc_bounds=(0, 100)):
```